cubelib/src/steps/tables.rs

```diff
@@ -156,10 +156,10 @@ impl PruningTables333 {
             match res {
                 Ok(v) => {
                     *val = Some(v);
-                    info!("Loaded {key} table from disk");
+                    debug!("Loaded {key} table from disk");
                 },
-                Err(e) => {
-                    error!("Error loading {key} table from disk: {e}");
+                Err(_) => {
+                    debug!("Could not load {key} table from disk");
```

Comment:
I was worried the first time I ran and saw this error message. But the program recovers fine and generates the tables, so the user doesn't ever have to know.

Reply:
Yeah, error level is definitely too high, especially for something that's expected the first time this runs. I do think that debug is too low, knowing that the code is currently generating tables rather than loading is useful. Maybe we print it as a warning, but make the message less scary?

Reply:
Ah, I just saw your definitions for what the levels mean. In this case I think it should be info. Knowing that the tables are being generated can be relevant for users, especially if we are talking about larger tables that take more than a few seconds to generate.

Reply:
Changed to `info`. Makes sense since the "Generating table ..." messages are also `info`